// Copyright 2020 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// Messages that are used as memcache values.
syntax = "proto2";

package recommender;

message ItemIds {
  repeated int64 item_id = 1;
}

message FeedItems {
  repeated FeedItem feed_item = 2;
}

message FeedItem {
  optional int64 item_id = 1;
  optional int64 published_timestamp_millis = 2;
  optional int64 retrieved_timestamp_millis = 3;
}
